Gems/LyShine/Code/Include/LyShine/Draw2d.h

```diff
@@ -25,6 +25,65 @@ class CDraw2d
     : public IDraw2d // LYSHINE_ATOM_TODO - keep around until gEnv->pLyShine is replaced by bus interface
     , public AZ::Render::Bootstrap::NotificationBus::Handler
 {
+public: // types
+
+    struct RenderState
+    {
+        RenderState()
+        {
+            m_blendState.m_enable = true;
+            m_blendState.m_blendSource = AZ::RHI::BlendFactor::AlphaSource;
+            m_blendState.m_blendDest = AZ::RHI::BlendFactor::AlphaSourceInverse;
+
+            m_depthState.m_enable = false;
+        }
+
+        AZ::RHI::TargetBlendState m_blendState;
+        AZ::RHI::DepthState m_depthState;
+    };
+
+    //! Struct used to pass additional image options.
+    //
+    //! If this is not passed then the defaults are used
+    struct ImageOptions
+    {
+        AZ::Vector3 color = AZ::Vector3(1.0f, 1.0f, 1.0f);
+        Rounding pixelRounding = Rounding::Nearest;
+        RenderState m_renderState;
+    };
+
+    //! Struct used to pass additional text options - mostly ones that do not change from call to call.
+    //
+    //! If this is not passed then the defaults below are used
+    struct TextOptions
+    {
+        AZStd::string   fontName;               //!< default is "default"
+        unsigned int    effectIndex;            //!< default is 0
+        AZ::Vector3     color;                  //!< default is (1,1,1)
+        HAlign          horizontalAlignment;    //!< default is HAlign::Left
+        VAlign          verticalAlignment;      //!< default is VAlign::Top
+        AZ::Vector2     dropShadowOffset;       //!< default is (0,0), zero offset means no drop shadow is drawn
+        AZ::Color       dropShadowColor;        //!< default is (0,0,0,0), zero alpha means no drop shadow is drawn
+        float           rotation;               //!< default is 0
+        bool            depthTestEnabled;       //!< default is false
```

Comment:
It might be a good idea to initialize these instead of relying on their constructors. Not sure if these values will need to get serialized but if they do, uninitialized values could result in producing inconsistent defaults each time.

Reply:
These are part of existing legacy code that was moved from IDraw2d.h to Draw2d.h, so I didn't want to change anything that didn't need to be changed at this time. Will keep it mind!